tests/test_api.py

```diff
@@ -714,6 +714,23 @@ def test_visualize_tangled_tree_layers(self, client, figure_type, data_model_jso
 
         assert response.status_code == 200
 
+    @pytest.mark.parametrize("component", ["Patient", "BulkRNA-seqAssay"])
```

Comment:
@GiaJordan could you use parameterize based on the example here: https://docs.pytest.org/en/6.2.x/parametrize.html#:~:text=42)%5D)%0Adef-,test_eval,-(test_input%2C I think that will simplify the code

Reply:
Sure @linglp, I can parameterize the expected responses as well